language/gpt3/megatron/dataset.py

```diff
@@ -0,0 +1,162 @@
+import os
+import sys
+
+sys.path.append(os.environ["MEGATRON_PATH"])
+from megatron.tokenizer import build_tokenizer
+from megatron.utils import get_ltor_masks_and_position_ids
+import torch
+import argparse
+import utils
+import json
+
+from argparse import Namespace
+
+
+PROMPT_DICT = {
+    "prompt_input": (
+        "{instruction}{input}"
```

Comment:
@pgmpablo157321 I think in Google's implementation, the instruction is just `summarize: ` instead of  the full `Summarize the ....` from GPTJ-6B. We can double-check next week again, but that's what I got from @morgandu
```suggestion
        "summarize: {input}"
```

Reply:
nvm, I think this has been resolved in download_cnndm.py. Sorry for the confusion.